README.md

```diff
@@ -75,16 +72,61 @@ By default, kola uses the `qemu` platform with the most recently built image
 
 #### kola run
 
-In order to provide internet access to QEMU instances; it's required to enable ipv4 forwarding on the host machine:
+##### Getting started with QEMU
 
+The easiest way to get started with `kola` is to run a `qemu` test.
+
+***requirements***:
+ - IPv4 forwarding (to provide internet access to the instance): `sudo sysctl -w net.ipv4.ip_forward=1`
+ - `dnsmasq` installed and present in the `$PATH`
```

Comment:
iptables too

Reply:
right - it's wrapped by the https://github.com/coreos/go-iptables lib.